c/pthread-ext/06_ticket_true-unreach-call.c

```diff
@@ -1,5 +1,8 @@
 extern void __VERIFIER_assume(int);
 extern void __VERIFIER_error() __attribute__ ((__noreturn__));
+int __global_lock;
+void __VERIFIER_atomic_begin() { __VERIFIER_assume(__global_lock==0); __global_lock=1; return; }
```

Comment:
Well, my understanding is that `__VERIFIER_atomic_begin/end` functions are the same as e.g. `__VERIFIER_nondet_int`. Meaning that no body should be provided for them, but only their declarations. That is at least what the SVCOMP specs say. Each verifier has to ensure to provide meaning for those functions based on the SVCOMP specs. Could you send an email to the SVCOMP mailing list to clarify this for sure?